exercises/isbn-verifier/description.md

```diff
@@ -1,4 +1,7 @@
-Check if a given ISBN-10 is valid.
+Given a number, check if a given ISBN-10 is valid.
+
+The [ISBN-10 verification process](http://www.hahnlibrary.net/libraries/isbncalc.html) requires use of a simple formula, and is mostly used to validate book identification
```

Comment:
Link to wikipedia and solve all your describe 'ISBN' problems: https://en.wikipedia.org/wiki/International_Standard_Book_Number